modules/yieldlabBidAdapter.js

```diff
@@ -142,6 +142,24 @@ export const spec = {
           }
         }
 
+        if (isNative(bidRequest, adType)) {
+          const url = `${ENDPOINT}/d/${matchedBid.id}/${bidRequest.params.supplyId}/?ts=${timestamp}${extId}${gdprApplies}${gdprConsent}${pvId}`
+          bidResponse.adUrl = url
+          bidResponse.mediaType = NATIVE
+          const nativeImageAsset = matchedBid.native.assets.find(e => e.id === 2) ? matchedBid.native.assets.find(e => e.id === 2).img : {url: '', w: 0, h: 0};
```

Comment:
Can we store the results of these `find` operations in some variables? Would improve readability and save some array iterations.